### DISCUSSION TITLE
Debugging application running inside the docker.

### DISCUSSION THREAD

**Discussion by dina75** [batect/batect#1484] [General]
How to run a docker with inspect argument for debugging application or equivalent. I was looking around similar option like --inspect=0.0.0.0:9229 which start the docker container locally with debugging port enabled, But no luck.

**Comment by charleskorn** [comment]
I'm unclear what you're looking for sorry @dina75 - could you please explain a little more about what you're trying to do? Or perhaps provide an example repo?

**Reply by dina75** [reply]
@charleskorn, Thanks for your kind response. I am after some sort of batect configuration for starting docker with debug port enabled for remote debugging the application running inside the docker container. using VScode/Intellij.

**Reply by charleskorn** [reply]
So you'd like to pass the `--inspect=0.0.0.0:9229` argument to the application in your container? Take a look at the documentation for [`command`](https://batect.dev/docs/reference/config/containers/#command).
